ironfish-cli/src/commands/wallet/multisig/dkg/create.ts

```diff
@@ -304,11 +308,11 @@ export class DkgCreateCommand extends IronfishCommand {
       errorOnDuplicate: true,
     })
 
-    input = await ui.inputPrompt('Enter the number of minimum signers', true)
-    const minSigners = parseInt(input)
-    if (isNaN(minSigners) || minSigners < 2) {
-      throw new Error('Minimum number of signers must be at least 2')
-    }
+    const minSigners = await ui.inputNumberPrompt(
```

Comment:
Do we still need to check whether `minSigners < 2`?

Reply:
added back